www/in-the-wild.html

```diff
@@ -183,10 +235,22 @@ <h4>Mapping</h4>
             <li>e.g. <a href="http://www.romaierioggi.it/nuova-pianta-di-roma-moderna-monaldini-1843/">Nuova Pianta di Roma Moderna – Monaldini 1843</a></li>
         </ul>
     </li>
+    <li><a href="http://heatmaps.nimblex.net/">SDR Heatmaps (radio frequencies)</a>
```

Comment:
I think this belongs in Scientific (and we can keep the mapping section just for geo stuff).